src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -177,7 +177,7 @@ public function getDuration()
 
         for ($i = 0; $i < $left; ++$i) {
             $index = $stopped + $i;
-            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow());
+            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow(), true);
```

Comment:
Except if the new arg is commented until 4.0, and implementations not having it trigger deprecations. `StopWatchPreciseEvent` could be a solution, but you still need a simple way to say you want to use it, higher in the Stopwatch API, without impacting existing code, as it may impact other consumers.